### DISCUSSION TITLE
Video background in Page Builder

### DISCUSSION THREAD

**Discussion by fvlasie** [breezefront/community#132] [1. Help] [answered]
Hello!

I am trying to apply a video background in Page Builder but it will not show when I am using Breeze Evolution. 
Does Breeze support video backgrounds?

If I test the same page content in Luma the video background works.....

**Comment by vovayatsyuk** [comment, marked as answer]
Not supported is the current version. Already supported in the dev branch, so it will be included in the next release.

**Reply by fvlasie** [reply]
Very good! Thank you!
